cmd/machine-config-operator/bootstrap.go

```diff
@@ -48,11 +48,17 @@ func init() {
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.destinationDir, "dest-dir", "", "The destination directory where MCO writes the manifests.")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.imagesConfigMapFile, "images-json-configmap", "", "ConfigMap that contains images.json for MCO.")
```

Comment:
This implies we can also drop the `images-json-configmap`, right? (And drop the "this overrides the image from --images-json-configmap" texts).